Tasks/Common/VstsAzureHelpers_/Utility.ps1

```diff
@@ -539,20 +540,18 @@ function Disconnect-AzureAndClearContext {
 
 function Disconnect-UsingAzModule {
     [CmdletBinding()]
-    param()
+    param(
+        [string]$restrictContext = 'False'
+    )
 
     if (Get-Command -Name "Disconnect-AzAccount" -ErrorAction "SilentlyContinue" -and CmdletHasMember -cmdlet Disconnect-AzAccount -memberName Scope) {	
```

Comment:
You do not have a need to do this check anymore.

Reply:
I think it will be better if we have it for now. Since Az module commands keep on changing. To avoid that, it can be considered as a safe option.